app/features/feedback/classifier/components/feedback-modal.jsx

```diff
@@ -27,17 +28,24 @@ class FeedbackModal extends React.Component {
 
   render() {
     const { messages, subjectViewerProps } = this.props;
+
     return (
       <ModalFocus className="classifier feedbackmodal">
         <Translate content="FeedbackModal.title" component="h2" />
         {subjectViewerProps && (<SubjectViewer {...subjectViewerProps} />)}
-        <ul>
-          {messages.map(message =>
-            <li key={Math.random()}>
-              {message}
-            </li>
-          )}
-        </ul>
+          {_.chain(messages).map((catMessages, cat) => {
+            return (<div key={Math.random()}>
+              <h3 key={Math.random()}>
+              {cat}
```

Comment:
This should probably be a `<Translate />` component, like `FeedbackModal.title` on line 34. Then store the actual category text in `app/locales/en.js`. That'll allow it to be translated.

Reply:
Good advice. Thanks.

Reply:
This change should still be incorporated.

Reply:
Let's still incorporate translation into this PR.